### PR TITLE
Remove .dev from pulp_rpm 3.25.0 version specifiers

### PR DESCRIPTION
**Pull request by dralley**
This reverts commit e2a10a2b16b0a5d13129c06d367f3ed4d7e980b5.

[noissue]

### CODE REVIEW

**Comment by dralley** [comment]
What's up with this error?  It seems sporadic, it happened on the other PR as well and went away w/o anything changing, but it's seemingly common..

File: tests/scripts/pulp_rpm/test_rpm_sync_publish.sh [hunk 1]
```diff
@@ -127,18 +127,18 @@ then
   expect_fail pulp rpm repository sync --repository "cli_test_rpm_repository" --sync-policy foobar
 fi
 
-if pulp debug has-plugin --name "rpm" --min-version "3.25.0.dev"
+if pulp debug has-plugin --name "rpm" --max-version "3.24.0"
```
Comment (mdellweg) [inline tests/scripts/pulp_rpm/test_rpm_sync_publish.sh:130]:
```suggestion
if pulp debug has-plugin --name "rpm" --specifier "<3.25.0"
```
Please check the version again...

Comment (dralley) [reply tests/scripts/pulp_rpm/test_rpm_sync_publish.sh]: I'm aware, but I keep hitting the same error unexpectedly (not the same as the sporadic one mentioned above), so I was trying a few different combinations

Comment (mdellweg) [reply tests/scripts/pulp_rpm/test_rpm_sync_publish.sh]: https://github.com/pulp/pulp-cli/actions/runs/7641882905/job/20820247714?pr=884#step:7:148
The nightly container is reporting to run `"rpm": "3.25.0.dev"` so we need to wait until that nightly container built with "3.26.0.dev" I would say.

**Review by ggainey — APPROVED** [review]

**Pull request merged**